Word_Search_79.java

```diff
@@ -0,0 +1,51 @@
+//Time Complexity : O(3^(word length))
+//Space Complexity : O(word length)
```

Comment:
Please rework the Time & Space.

Reply:
Time: O(n*m * 3^l), one recursive call can be made for each space in the 2-D matrix.
Space: O(n*m), recursive stack space.